README.md

```diff
@@ -41,6 +41,29 @@ Simply pass `searchable: false` to your content definitions and Alchemy will not
     searchable: false
     default: 'This is my secret password.'
 ```
+  
+### Configure Behavior
+         
+Configure the gem in an initializer. The default configurations are:
+
+```ruby
+   Alchemy::PgSearch.config = {
+      paginate_per: 10, # amount of results per page
+  }
+```
```

Comment:
The indentation is a bit off here

```suggestion
```ruby
Alchemy::PgSearch.config = {
  paginate_per: 10, # amount of results per page
}
```
```